specification/network/resource-manager/Microsoft.Network/2017-09-01/examples/ApplicationSecurityGroupCreate.json

```diff
@@ -0,0 +1,37 @@
+{
+  "parameters": {
+    "api-version": "2017-09-01",
+    "subscriptionId": "subid",
+    "resourceGroupName": "rg1",
+    "applicationSecurityGroupName": "test-asg",
+    "location": "westus",
+    "parameters": {
+      "properties": {
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "name": "test-asg",
+        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/applicationSecurityGroups/test-asg",
+        "type": "Microsoft.Network/applicationSecurityGroups",
+        "location": "westus",
+        "properties": {
```

Comment:
The definition of "applicationSecurityGroup" has one more property named "resourceGuid". This is missing in the example for 201 and 200 response. Based on the description it seems that this property will always be present and it will be returned by the server. Can the user provide this resourceGuid while creating the applicationSecurityGroup? If no then should this property be marked as readOnly: true?

Reply:
Correct. I added the missing field and marked it as read-only.